corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py

```diff
@@ -47,6 +47,7 @@ def render_case(case, options):
     from corehq.apps.hqwebapp.templatetags.proptable_tags import get_tables_as_rows, get_definition
     case = wrapped_case(case)
     timezone = options.get('timezone', pytz.utc)
+    timezone = timezone.localize(datetime.datetime.now()).tzinfo
     _get_tables_as_rows = partial(get_tables_as_rows, timezone=timezone)
```

Comment:
I think it's worth figuring out the daylight savings question. I thought we thought we knew how to use timezones, but apparently not entirely. I think the best outcome would be to have a single function that we can use that's clearly documented and strict, and then to have all timezone localizations use that function. (Even if it's just one line, still worth enshrining it in a function as the one right way to do it.) Second best outcome would be to document somewhere what the definitely correct way to convert timezone naive (implicit UTC) datetimes to timezone aware datetimes is, and link everyone to that, and make it required reading.